.vscode/extensions.json

```diff
@@ -0,0 +1,11 @@
+{
+	// List of extensions which should be recommended for users of this workspace.
```

Comment:
Are comments actually supported like this? They aren't in normal json...

Reply:
This is specific  to VS Code according to [docs](https://vscode-docs.readthedocs.io/en/stable/languages/json/)